tailscale/client.go

```diff
@@ -333,6 +332,62 @@ func (c *Client) DNSNameservers(ctx context.Context) ([]string, error) {
 	return resp["dns"], nil
 }
 
+// SplitDnsRequest is a map from domain names to a list of nameservers.
+type SplitDnsRequest map[string][]string
```

Comment:
Document on the type what the structure is....  keys are the domain to set split dns for and values are the slice of nameservers.